eventmesh-common/src/main/java/org/apache/eventmesh/common/Constants.java

```diff
@@ -28,6 +28,8 @@ public class Constants {
 
     public static final String DATE_FORMAT_DEFAULT = "yyyy-MM-dd HH:mm:ss";
 
+    public static final String DATA_CONTENT_TYPE = "DataContentType";
```

Comment:
<img width="534" alt="image" src="https://user-images.githubusercontent.com/29346818/228104739-6f6ec65a-4eff-428a-8bd6-18a1f1a5ac6c.png">

The naming of protocol attributes can be unified.